src/results/results.jl

```diff
@@ -167,6 +208,34 @@ function combine_results(p::REoptInputs, bau::Dict, opt::Dict, bau_scenario::BAU
     opt["ElectricLoad"]["bau_critical_load_met"] = bau_scenario.outage_outputs.bau_critical_load_met
     opt["ElectricLoad"]["bau_critical_load_met_time_steps"] = bau_scenario.outage_outputs.bau_critical_load_met_time_steps
 
+    # emissions reductions
+    opt["Site"]["lifecycle_emissions_reduction_CO2_pct"] = (
+        bau["Site"]["lifecycle_emissions_tCO2"] - opt["Site"]["lifecycle_emissions_tCO2"]
+    ) / bau["Site"]["lifecycle_emissions_tCO2"]
+
+    # breakeven cost of CO2 (to make NPV = 0)
+    # first, remove climate costs from the output NPV, if they were previously included in LCC/NPV calcs:
+    npv_without_modeled_climate_costs = opt["Financial"]["npv"]
+    if p.s.settings.include_climate_in_objective == true
+        npv_without_modeled_climate_costs -= (bau["Site"]["lifecycle_emissions_cost_CO2"] - opt["Site"]["lifecycle_emissions_cost_CO2"])
+    end
+    # we want to calculate the breakeven year 1 cost of CO2 (usd per tonne) that would yield an npv of 0, holding all other inputs constant
+    # (back-calculating using the equation for m[:Lifecycle_Emissions_Cost_CO2] in "add_lifecycle_emissions_calcs" in emissions_constraints.jl)
+    if npv_without_modeled_climate_costs >= 0 # if the system is cost effective (NPV >= 0) without considering any cost of CO2, no breakeven value is reported 
+        opt["Financial"]["breakeven_cost_of_emissions_reduction_per_tCO2"] = nothing
+    else
+        breakeven_cost_denominator = p.pwf_emissions_cost["CO2_grid"] * (
+            bau["ElectricUtility"]["year_one_emissions_tCO2"] - opt["ElectricUtility"]["year_one_emissions_tCO2"]
+        ) + p.pwf_emissions_cost["CO2_onsite"] * (
+            bau["Site"]["year_one_emissions_from_fuelburn_tCO2"] - opt["Site"]["year_one_emissions_from_fuelburn_tCO2"] 
+        )
+        if breakeven_cost_denominator != 0.0
+            opt["Financial"]["breakeven_cost_of_emissions_reduction_per_tCO2"] = -1 * npv_without_modeled_climate_costs / breakeven_cost_denominator
+        else
+            opt["Financial"]["breakeven_cost_of_emissions_reduction_per_tCO2"] = nothing
```

Comment:
Maybe just exclude the "breakeven_cost_of_emissions_reduction_per_tCO2" key instead of returning a `nothing` value?

Reply:
Defaulted to 0.0 in https://github.com/NREL/REopt.jl/commit/7c7fba7d1e0da2e4b752e23f0c0bd5b3a58ba9fe

Reply:
I've now excluded it